record/record.go

```diff
@@ -100,7 +100,7 @@ func PackageRecords(records []LogRecord) ([]PackagedRecords, error) {
 	// TODO Check Ian/Brian: I do believe that this should be compresssedData.Len(), let's confirm it before changing.
 	compressedSize := int64(compressedData.Cap())
 	if compressedSize >= maxPacketSize && len(records) == 1 {
-		log.Printf("[ERROR] Can't compress record below required maximum packet size and it will be discarded. Record: %v", records[0])
```

Comment:
Hey, I'm not sure I understand why we changed this.

Reply:
Hey! The reason was because it was printing a huge (and useless) log representing the compressed record which could not be sent due to being too big. This unnecessarily polluted the unit test results.

Reply:
Make sense, maybe the question is if it belongs to this PR? 😈  hehehe